docs/en/benchmark/deepseekv3.md

```diff
@@ -0,0 +1,113 @@
+# DeepSeekV3 Support
+
+| max bsz | prompt no | input-len | output-len | input token throughput (tok/s) | output token throughput (tok/s) | RPS  |
```

Comment:
Let's set up a secondary directory, with the benchmark for each version as a separate section.